scripts/verify-git-tag

```diff
@@ -1,11 +1,12 @@
-#!/bin/bash
+#!/bin/bash --
 
 # Usage: $0 <source-dir> [<ref>]
 # Example refs:
 #  master
 #  HEAD
 #  mainstream/master
 # Default ref: HEAD
+: "${KEYRING_DIR_GIT=}" "${NO_CHECK=}" "${DEBUG=}" "${VERBOSE=0}"
```

Comment:
Looking at the other changes in this script, it seems you wanted to add `set -eu` here too.

Reply:
I had not audited the top part of the script for behaving well under `set -euo pipefail`, but I will.